java/org/apache/coyote/AbstractProtocol.java

```diff
@@ -262,6 +267,17 @@ public void setConnectionLinger(int connectionLinger) {
         endpoint.setConnectionLinger(connectionLinger);
     }
 
+    // ------------------------------------------------ HTTP specific properties
+    // ----------------------------------------- queried by StandardContextValve
+    public void setContinueHandlingResponsePolicy(String value) {
+        continueHandlingResponsePolicy = Enum.valueOf(ContinueHandlingResponsePolicy.class, value);
```

Comment:
I think you need a `trim()` and a `.toUpperCase(Locale.ENGLISH)` here

Reply:
I've added these, thanks for catching!